### ISSUE TITLE
mat-select aria-label

### ISSUE THREAD

**Issue by kekraft** [angular/material2#7238]
Bug:
mat-select should have the aria-label of the placeholder or mat-select-trigger read by default. Currently, with Chrome Vox the default behavior is to get the role 'listbox' read.

Expected behavior:
By default, mat-select should have the aria-label of the mat-select or mat-select-trigger contents read.

Current behavior:
mat-select has 'listbox' role shadowing the aria-label.

To reproduce:

Go to: https://material.angular.io/components/select/examples
Enable Chrome Vox.
Tab to the 'Favorite food' dropdown and get 'listbox' by the reader.

**Comment by josephperrott** [comment]
Closing as duplicate of #5220
